README.md

```diff
@@ -1,2 +1,79 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Collaborative Project: The Next Web
+
+> This is a microverse student project for working with hierarchy of elements and responsive design
+
+![screenshot](https://github.com/Gnwin/Godwin-Eraani-Next-Web-project/blob/nextweb/assets/images/Screenshotnewtnw.png)
+
+> This project clones the thenextweb.com article page interface design.
+> The project includes the header, the main content, footer, media queries.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/Gnwin/Godwin-Eraani-Next-Web-project/nextweb/index.html)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+> Browser
+> Text editor of your choice.
+> npm has to be installed in your system.
+> install linters on your IDE.
+
+### Setup
+
+> `git clone https://github.com/Gnwin/Godwin-Eraani-Next-Web-project.git`
+
+### Usage
+
+> Check the nextweb branch.
+> The project was finished up on the nextweb branch
+> My partner contributed hers on nextbranch branch.
+> Open the index.html from your browser.
+
+### Run tests
+
+Run `npx html-validator-cli --quiet --file index.html` to run the validator over the `index.html` file.
+
+The `--quiet` will only list `errors`. If you also want to see the warnings use the `--verbose` flag instead.
+As an alternative, you can also install the `html5validator` using `pip`:
+
+```
+pip install --user html5validator
+html5validator index.html
+```
+
+## Authors
+
+👤 **Godwin Nwachukwu**
+
+- Github: [@Gnwin](https://github.com/Gnwin)
```

Comment:
If you have more social links such as `linkedin` or `twitter`, you should have added them here as well.